apps/aot/src/app/about/page.tsx

```diff
@@ -94,17 +92,17 @@ export default function AboutPage() {
             >
               TypeScript Playground
             </a>{' '}
-            preferred over our editor(at least for now 🤐). Just copy the code from the test
-            section, start coding, and when you nail it, pop your code (except the tests) back into
-            our editor.
+            preferred over our editor (at least for now 🤐). You can copy the code from the test
```

Comment:
> Just

https://tipsforteachers.substack.com/p/59-stop-saying-the-word-just?open=false#%C2%A7a-tip-to-try-in-class-this-week